azure-local/manage/virtual-machine-operations.md

```diff
@@ -16,12 +16,11 @@ This article discusses the most common operations for Azure Local virtual machin
 
 ## Overview
 
-Azure Local can host various types of VMs including Arc-enabled servers, unmanaged VMs, and VMs enabled by Azure Arc. When comparing these options, the VMs enabled by Azure Arc provide the highest level of management capabilities in Azure portal, second only to [native Azure VMs](/azure/azure-local/concepts/compare-vm-management-capabilities).
+Azure Local can host various types of VMs including unmanaged VMs, Arc-enabled servers, and VMs enabled by Azure Arc. When comparing these options, the VMs enabled by Azure Arc provide the highest level of management capabilities in Azure portal, second only to [native Azure VMs](/azure/azure-local/concepts/compare-vm-management-capabilities).
 
 While Azure Local VMs are designed to be managed through the Azure control plane and have numerous management features within the portal, on-premises local tools offer a broader range of capabilities. These tools include System Center Virtual Machine Manager (SCVMM), Failover Cluster Manager, Hyper-V Manager, and Windows Admin Center. Many IT admins use these tools to manage their on-premises VMs.
 
-When IT admins manage Azure Local VMs using the same tools and scripts as those for on-premises VMs, this results in synchronization issues between the Azure Local VMs and the portal. Though the operations listed under [Supported operations for local tools](#using-local-tools) don't affect the management of Azure Local VMs, the changes aren't reflected in the Azure portal.
-
+When IT admins manage Azure Local VMs using the same tools and scripts as those for on-premises VMs, this may results in synchronization or more severe problems between the Azure Local VMs and the portal. 
```

Comment:
We don't use "may", also there is a slight inaccuracy and then there is need to disambiguate. Change to: When IT admins manage Azure Local VMs using the same tools and scripts as those for on-premises VMs, they could encounter synchronization issues or other severe issues between the Azure Local VMs and the portal.

Reply:
Sounds great!